web/src/settings_users.js

```diff
@@ -218,6 +218,12 @@ function bot_info(bot_user_id) {
     info.bot_owner_id = owner_id;
     info.user_role_text = people.get_user_type(bot_user_id);
 
+    if (bot_user.avatar_url) {
+        info.img_src = bot_user.avatar_url;
+    } else {
+        info.img_src = people.medium_avatar_url_for_person(bot_user);
```

Comment:
Simplified the if-else block and replaced `medium_avatar_url_for_person` with `small_avatar_url_for_person`. I had this block because the `user_object/bot_object` already had an `avatar_url` present, and in cases where this might be empty, I used `medium_avatar_url_for_person`.